rust/Cargo.toml

```diff
@@ -19,7 +19,7 @@ aptos-moving-average = { path = "moving-average" }
 
 ahash = { version = "0.8.7", features = ["serde"] }
 anyhow = "1.0.62"
-aptos-protos = { git = "https://github.com/aptos-labs/aptos-core.git", tag = "aptos-node-v1.12.1" }
+aptos-protos = { git = "https://github.com/aptos-labs/aptos-core.git", rev = "d76b5bb423b78b2b9affc72d3853f0d973d3f11f" }
```

Comment:
Not a blocker but any reason we're not using a tag now?

Reply:
it's in devnet; tag will only be available for testnet/mainet release candidate.

1. we can change back to tag once it's in testnet.
2. we can request devnet tag to make life easier.